modules/rostests/kmtests/tcpip/connect.c

```diff
@@ -229,11 +234,24 @@ TestTcpConnect(void)
     ok_eq_long(ReturnInfo.UserDataLength, 0);
     ok_eq_pointer(ReturnInfo.UserData, NULL);
 
-    ok_eq_long(ReturnAddress.TAAddressCount, 0);
-    ok_eq_hex(ReturnAddress.Address[0].AddressType, 0);
-    ok_eq_hex(ReturnAddress.Address[0].AddressLength, 0);
-    ok_eq_hex(ReturnAddress.Address[0].Address[0].sin_port, 0);
-    ok_eq_hex(ReturnAddress.Address[0].Address[0].in_addr, 0);
+    if (OSVersionInfo.dwMajorVersion < 6)
```

Comment:
This is not needed, just fix it for NT5 unconditionally.

Reply:
Reverted.